### ISSUE TITLE
1.3.0: uncaught exception in Web UI: invalid byte sequence in US-ASCII

### ISSUE THREAD

**Issue by jessereynolds** [flapjack/flapjack#802]
Requesting / from the web gateway can return a blank page and log the following exception. 

```
2015-02-26T22:32:42.549461+11:00 [INFO] :: flapjack-web :: GET /
2015-02-26T22:32:43.612270+11:00 [ERROR] :: flapjack-web :: invalid byte sequence in US-ASCII
2015-02-26T22:32:43.613894+11:00 [ERROR] :: flapjack-web :: /opt/flapjack/embedded/lib/ruby/gems/2.1.0/gems/flapjack-1.3.0/lib/flapjack/data/entity_check.rb:151:in `split'
/opt/flapjack/embedded/lib/ruby/gems/2.1.0/gems/flapjack-1.3.0/lib/flapjack/data/entity_check.rb:151:in `block in count_current_failing'
/opt/flapjack/embedded/lib/ruby/gems/2.1.0/gems/flapjack-1.3.0/lib/flapjack/data/entity_check.rb:150:in `count'
/opt/flapjack/embedded/lib/ruby/gems/2.1.0/gems/flapjack-1.3.0/lib/flapjack/data/entity_check.rb:150:in `count_current_failing'
/opt/flapjack/embedded/lib/ruby/gems/2.1.0/gems/flapjack-1.3.0/lib/flapjack/gateways/web.rb:492:in `check_stats'
/opt/flapjack/embedded/lib/ruby/gems/2.1.0/gems/flapjack-1.3.0/lib/flapjack/gateways/web.rb:160:in `block in <class:Web>'
/opt/flapjack/embedded/lib/ruby/gems/2.1.0/gems/sinatra-1.4.5/lib/sinatra/base.rb:1603:in `call'
...
```

**Comment by jessereynolds** [comment]
I think this started when I sent through an event with entity `¯\_(ツ)_/¯`

**Comment by ali-graham** [comment]
What's the correct thing to do if Flapjack's running in an environment with ASCII encoding but receives Unicode-encoded data? Presumably we should set a UTF-8 encoding initially and then enforce that all input channels are converted to it if required... assuming that will work.

**Comment by jessereynolds** [comment]
> What's the correct thing to do if Flapjack's running in an environment with ASCII encoding but receives Unicode-encoded data?

Good question. Try and sanitise? Eg delete invalid chars, or set them to `_` or some such? 

> Presumably we should set a UTF-8 encoding initially and then enforce that all input channels are converted to it if required...

Do we just need to set an environment variable in the init script so UTF-8 is default?

**Comment by jessereynolds** [comment]
Fixed in #805
